codecov.yml

```diff
@@ -1 +1,6 @@
-comment: false
\ No newline at end of file
+comment: false
+ignore:
+  - "lib/**/*engine.rb"
```

Comment:
Shouldn't these path include something like `decidim-*`? I'm not sure how this works though!

Reply:
Proposa'ls engine is in `lib/decidim/proposals/engine.rb` so the ignore path can be `lib/decidim/**/engine.rb`, not sure if it's worth changing it

Reply:
I meant at the begining of the path!

Reply:
`codecov/project — 88.66% (+9.75%)` seems to be already incrementing. Anyway what you propose seems logical so I'll try it and see what happens